source/retryable-writes/tests/README.rst

```diff
@@ -169,6 +169,18 @@ Each YAML file has the following keys:
     the default is all topologies (i.e. ``["single", "replicaset", "sharded",
     "load-balanced"]``).
 
+  - ``serverless``: Optional string. Whether or not the test should be run on
```

Comment:
Just to confirm, this is being added with the assumption that it _may_ be needed down the line to exclude certain tests? As-is, it seems that all retryable writes tests are expected to pass on serverless.

Reply:
Correct, no retryable writes tests currently need this. I have added the `serverless` field since these are run as part of serverless.